src/src/config.h

```diff
@@ -0,0 +1,40 @@
+#pragma once 
+
+#include <Arduino.h>
+#include "targets.h"
+#include "elrs_eeprom.h"
+
+#define CONFIG_VERSION          1
+#define RESERVED_EEPROM_SIZE    512
+
+typedef struct {
+    uint32_t    version;
+    uint32_t    rate;
+    uint32_t    tlm;
+    uint32_t    power;
+    bool        modified;
```

Comment:
uh, do we want the modified flag to be written to the flash? This feels like it should be just a runtime thing that would be a seperate bool in the class

Reply:
Good point. This should be split out.